assembly/env/env.ts

```diff
@@ -33,6 +33,11 @@ export namespace env {
   @external("massa", "assembly_script_caller_has_write_access")
   export declare function callerHasWriteAccess(): bool;
 
+  @inline
+  export function isDeployingContract(): bool {
+    return callerHasWriteAccess();
+  }
```

Comment:
env is only containing external function linker. This function has nothing to do here.

Can you move it to context.ts ?

Furthermore, can you document it ?

Reply:
done